src/Analysis/Engine/Impl/Interpreter/Ast/AstModuleResolution.cs

```diff
@@ -205,6 +205,12 @@ internal sealed class AstModuleResolution {
                 } catch (OperationCanceledException) {
                     _log?.Log(TraceLevel.Error, "ImportTimeout", name, "ImportFromSearchPaths");
                     return TryImportModuleResult.Timeout;
+                } catch (Exception ex) when (
+                    ex is IOException // FileNotFoundException, DirectoryNotFoundException, PathTooLongException, etc 
+                    || ex is UnauthorizedAccessException
+                ) {
+                    _log?.Log(TraceLevel.Error, "ImportException", name, "ImportFromSearchPaths", ex.GetType().Name, ex.Message);
+                    return TryImportModuleResult.ModuleNotFound;
```

Comment:
You may choose `RetryNeeded` return in case of `UnauthorizedAccessException` since the exception may be a conflict when some other process is locking the file.

Reply:
@jakebailey - could you address the above and then we can merge.

Reply:
Oops, forgot I had sent this PR. Will do.

Reply:
Done.